src/frontend/src/icp-eth/components/fee/FeeAmountDisplay.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+	import { nonNullish } from '@dfinity/utils';
+	import { fade, slide } from 'svelte/transition';
+	import { formatToken } from '$lib/utils/format.utils';
+	import { EIGHT_DECIMALS } from '$lib/constants/app.constants';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
+	import { balancesStore } from '$lib/stores/balances.store.js';
+	import { BigNumber } from '@ethersproject/bignumber';
+	import type { TokenId } from '$lib/types/token';
+	import { onDestroy } from 'svelte';
+
+	export let fee: BigNumber;
+	export let feeSymbol: string;
+	export let feeTokenId: TokenId;
+
+	const balance = nonNullish($balancesStore)
+		? $balancesStore[feeTokenId]?.data ?? BigNumber.from(0n)
+		: undefined;
+
+	let insufficientFeeFunds = false;
+	let timer: NodeJS.Timeout | undefined;
+
+	$: {
+		timer = setTimeout(() => {
```

Comment:
not sure what's the purpose of this timeout?

Reply:
Ah! so, I wanted to have a sort of delay between the two different animations: the ideal flow is that the fee updates and right after the error slides in, adjusting the height of the modal in a smooth way; 

otherwise they would fade in together and the modal will auto-adjust the height in a static way, with no transitions

Reply:
I see, could that be also solved by specifying a `delay` to one of the two Svelte animations?

Reply:
I tried that at first, but it was not working. Maybe my rendering was already messed up. I will try again in the PR where I separate that part.

Reply:
ok